<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="reenas-MacBook-Pro.local" ignored="1" name="Test" tests="2" failures="0" timestamp="13 Jun 2017 15:30:59 GMT" time="40.9" errors="0">
  <testcase name="@BeforeTest launchapp" time="0.0" classname="com.test.automation.testBase.TestBase">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &amp;apos;2.53.0&amp;apos;, revision: &amp;apos;35ae25b1534ae328c771e0856c93e187490ca824&amp;apos;, time: &amp;apos;2016-03-15 10:43:46&amp;apos;
System info: host: &amp;apos;reenas-MacBook-Pro.local&amp;apos;, ip: &amp;apos;192.168.2.70&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;10.12.5&amp;apos;, java.version: &amp;apos;1.8.0_91&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'reenas-MacBook-Pro.local', ip: '192.168.2.70', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.5', java.version: '1.8.0_91'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:158)
	at com.test.automation.testBase.TestBase.launchapp(TestBase.java:97)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to localhost:5555 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)
	... 27 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 40 more
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest launchapp -->
  <testcase name="@BeforeClass setUp" time="0.0" classname="com.test.automation.homePage.VerifyLoginWithValidCredentials">
    <skipped/>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.test.automation.homePage.VerifyLoginWithValidCredentials">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@AfterClass endTest" time="0.0" classname="com.test.automation.homePage.VerifyLoginWithValidCredentials">
    <skipped/>
  </testcase> <!-- @AfterClass endTest -->
  <testcase name="@BeforeClass setUp" time="0.0" classname="com.test.automation.settingsPage.VerifyPostCodePresent">
    <skipped/>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.test.automation.settingsPage.VerifyPostCodePresent">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@AfterClass endTest" time="0.0" classname="com.test.automation.settingsPage.VerifyPostCodePresent">
    <skipped/>
  </testcase> <!-- @AfterClass endTest -->
  <testcase name="verifyLoginWithValidCredentials" time="0.001" classname="com.test.automation.homePage.VerifyLoginWithValidCredentials">
    <skipped/>
  </testcase> <!-- verifyLoginWithValidCredentials -->
  <testcase name="verifyPostCodePresent_Test" time="0.001" classname="com.test.automation.settingsPage.VerifyPostCodePresent">
    <skipped/>
  </testcase> <!-- verifyPostCodePresent_Test -->
  <testcase name="@BeforeTest launchapp" time="0.0" classname="com.test.automation.testBase.TestBase">
    <ignored/>
  </testcase> <!-- @BeforeTest launchapp -->
</testsuite> <!-- Test -->
